.script/tests/detectionTemplateSchemaValidation/ValidConnectorIds.json

```diff
@@ -37,7 +37,9 @@
   "Citrix",
   "CitrixWAF",
   "CloudflareDataConnector",
+  "ContrastProtect",
```

Comment:
which connector is it?

Reply:
I have updated these data connector id's present in [solution folder.](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions)